api/envoy/config/listener/v3/udp_listener_config.proto

```diff
@@ -33,8 +33,18 @@ message UdpListenerConfig {
 
   // Configuration for QUIC protocol. If empty, QUIC will not be enabled on this listener. Set
   // to the default object to enable QUIC without modifying any additional options.
-  // [#not-implemented-hide:]
-  // [#comment:Unhide when QUIC alpha is announced with other docs.]
+  //
+  // .. note::
+  //   QUIC support is currently alpha and should be used with caution.
+  //
+  //   For known outstanding issues before Envoy QUIC support is GA, you can
+  //   track https://github.com/envoyproxy/envoy/labels/quic-mvp for example
+  //   QUIC does not currently support in place filter chain updates, so users
```

Comment:
nit: I feel like 'in-place' is how I am accustomed to seeing this phrase in the context of software. (As opposed to, say, "having everything in place"). Maybe? WDYT?